Packs/CrowdStrikeFalcon/ReleaseNotes/2_4_10.md

```diff
@@ -0,0 +1,10 @@
+#### Integrations
+
+##### CrowdStrike Falcon
+
+- Updated the Docker image to: *demisto/py3-tools:1.0.0.5475267*.
+
+- Updated the CrowdStrikeFalcon integration to remove support for legacy API.
```

Comment:
No need customer facing updates. You can just add something like code improvment.